pkg/api/validation/validation.go

```diff
@@ -58,6 +59,7 @@ var DNS1123LabelErrorMsg string = fmt.Sprintf(`must be a DNS label (at most %d c
 var DNS952LabelErrorMsg string = fmt.Sprintf(`must be a DNS 952 label (at most %d characters, matching regex %s): e.g. "my-name"`, validation.DNS952LabelMaxLength, validation.DNS952LabelFmt)
 var pdPartitionErrorMsg string = IntervalErrorMsg(0, 255)
 var PortRangeErrorMsg string = IntervalErrorMsg(0, 65536)
+var IdRangeErrorMsg string = fmt.Sprintf(`UIDs and GIDs must be in range %d-%d`, 0, math.MaxInt32)
```

Comment:
IntervalErrorMsg(0, math.MaxInt32), "must be greater than %d and less than %d" means can't equal 0 and math.MaxInt32. For uid and gid it can equal it. Would you like IntervalErrorMsg(-1, math.MaxInt32+1) ?